ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Permissions.resource

```diff
@@ -102,3 +104,51 @@ Is ${username} In The Permissions Table
     ${present}=    Run Keyword And Return Status
     ...    Page Should Contain Element       xpath=//tr[td[@data-label="Username"]//*[text()="${username}"]]
     RETURN    ${present}
+
+Set RHODS Admins Group Empty Group
+    [Documentation]     Sets the "adminGroups" field in "odh-dashboard-config" ConfigMap
+    ...                 to the given empty group (i.e., with no users)
+    Set Access Groups Settings    admins_group=${CUSTOM_EMPTY_GROUP}
+    ...    users_group=${STANDARD_SYSTEM_GROUP}
+
+Set RHODS Admins Group To system:authenticated    # robocop:disable
+    [Documentation]    Sets the "adminGroups" field in "odh-dashboard-config" ConfigMap
+    ...    to the system:authenticated group
+    Set Access Groups Settings    admins_group=system:authenticated
+    ...    users_group=${STANDARD_SYSTEM_GROUP}
+
+Set RHODS Admins Group To Inexistent Group
+    [Documentation]    Sets the "adminGroups" field in "odh-dashboard-config" ConfigMap
+    ...    to the given inexistent group
+    Set Access Groups Settings    admins_group=${CUSTOM_INEXISTENT_GROUP}
+    ...    users_group=${STANDARD_SYSTEM_GROUP}
+
+Set RHODS Users Group Empty Group
+    [Documentation]    Sets the "allowedGroups" field in "odh-dashboard-config" ConfigMap
+    ...    to the given empty group (i.e., with no users)
+    Set Access Groups Settings    admins_group=${STANDARD_ADMINS_GROUP}
+    ...    users_group=${CUSTOM_EMPTY_GROUP}
+
+Set RHODS Users Group To rhods-users    # robocop:disable
+    [Documentation]    Sets the "allowedGroups" field in "odh-dashboard-config" ConfigMap
+    ...    to the rhods-users group
+    Set Access Groups Settings    admins_group=${STANDARD_ADMINS_GROUP}
+    ...    users_group=${STANDARD_USERS_GROUP}
+
+Set RHODS Users Group To Inexistent Group
+    [Documentation]    Sets the "allowedGroups" field in "odh-dashboard-config" ConfigMap
+    ...    to the given inexistent group
+    Set Access Groups Settings    admins_group=${STANDARD_ADMINS_GROUP}
+    ...    users_group=${CUSTOM_INEXISTENT_GROUP}
+
+Set Default Groups And Check Logs Do Not Change
+    [Documentation]    Teardown for ODS-1408 and ODS-1494. It sets the default configuration of "odh-dashboard-config"
+    ...    ConfigMap and checks if no new lines are generated in the logs after that.
+    [Arguments]    ${delete_group}=${FALSE}
+    ${lengths_dict}=    Get Lengths Of Dashboard Pods Logs
+    Set Access Groups Settings    admins_group=${STANDARD_ADMINS_GROUP}
+    ...    users_group=${STANDARD_SYSTEM_GROUP}
+    Logs Of Dashboard Pods Should Not Contain New Lines  lengths_dict=${lengths_dict}
+    IF  "${delete_group}" == "${TRUE}"
+        Delete Group    group_name=${CUSTOM_EMPTY_GROUP}
+    END
```

Comment:
I think these keywords should go in `ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource`. 

`ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Permissions.resource` is supposed to contain keyword for handling the Permissions of the DS projects, which is different from Dashboard permissions

Reply:
I moved them from `ods_ci/tests/Tests/400__ods_dashboard/400__ods_dashboard.robot` :
https://github.com/red-hat-data-services/ods-ci/pull/1101/files#diff-be67337b3f53b14968b105e32d5a8a2769771595618bb421553a393adb1373f6

Into `ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Permissions.resource`
(And just added few more similar keywords), because they all uses `Set Access Groups Settings`, and it seems more relevant.

Reply:
As you suggested, I moved these Keywords into `ODHDashboardSettings.resource`